homeassistant/helpers/state.py

```diff
@@ -124,9 +123,7 @@ def reproduce_state(hass, states, blocking=False):
 @asyncio.coroutine
 def async_reproduce_state(hass, states, blocking=False):
     """Reproduce given state."""
-    if isinstance(states, State):
-        states = [states]
-
+    states = [state for entities in states for state in entities]
```

Comment:
We should move this to the homeassistant scene platform and not change the type of the states argument for this function.